lib/build/tasks.js

```diff
@@ -56,7 +56,10 @@ function sassExtensions () {
   fileContents += extensions.getFileSystemPaths('sass')
     .map(filePath => `@import "${filePath.split(path.sep).join('/')}";`)
     .join('\n')
-  fs.writeFileSync(path.join('lib', 'extensions', '_extensions.scss'), fileContents)
+  const extensionsDir = path.join('.tmp', 'sass')
+  fse.ensureDirSync(extensionsDir, { recursive: true })
+  fs.writeFileSync(path.join(extensionsDir, '.gitignore'), '*')
```

Comment:
Oh nice, just noticed this 🙌 

Could this .gitignore go in `.tmp` though, instead of `.tmp/sass`? Seems neater to me.